src/types/onyx/ReimbursementAccount.ts

```diff
@@ -35,6 +35,25 @@ type ACHData = Partial<BeneficialOwnersStepProps & CompanyStepProps & RequestorS
 
     /** Policy ID of the workspace the bank account is being set up on */
     policyID?: string;
+
+    /** Weather Onfido setup is complete */
+    isOnfidoSetupComplete?: boolean;
+
+    mask?: string;
+
+    isSavings?: string;
+
+    plaidAccountID?: string;
+
+    plaidAccessToken?: string;
+
+    selectedPlaidAccountID?: string;
+
+    amount1?: string;
+
+    amount2?: string;
+
+    amount3?: string;
```

Comment:
I think you can reuse existing types, for example
https://github.com/Expensify/App/blob/104565b99a82751af0dd397c910f38958b6cb39a/src/types/form/ReimbursementAccountForm.ts#L112-L120